autogpt_platform/backend/backend/server/v2/store/media.py

```diff
@@ -0,0 +1,100 @@
+import logging
+import os
+import uuid
+
+import dotenv
+import fastapi
+from google.cloud import storage
+
+import backend.server.v2.store.exceptions
+
+dotenv.load_dotenv()
```

Comment:
Please use `backend.util.settings.Config` for this

Reply:
Done